docs/auth/android.md

```diff
@@ -15,23 +15,26 @@ dependencies {
 
 ## Install the RNFirebase Authentication package
 
+**Note**: This is for React Native 6.0+ - for earlier versions of RN please refer to the [previous version of this documentation](https://github.com/invertase/react-native-firebase-docs/blob/8c36d9df1a35e993f5d4126c33448256b9c5b682/docs/auth/android.md).
```

Comment:
I am sorry for being pedantic, but doing the little things right in docs is important :-)

```suggestion
**Note**: This is for react-native 0.60+ - for earlier versions of react-native please refer to the [previous version of this documentation](https://github.com/invertase/react-native-firebase-docs/blob/8c36d9df1a35e993f5d4126c33448256b9c5b682/docs/auth/android.md).
```